README.md

```diff
@@ -510,6 +511,40 @@ data:
 You're welcome to raise issues related to feature requests, bugs, or give us general feedback on this project's GitHub Issues page. 
 The SAP BTP service operator project maintainers will respond to the best of their abilities. 
 
+[Back to top](#sap-business-technology-platform-sap-btp-service-operator-for-kubernetes)
+
+## Deleting the Operator
+
+To delete the operator, run the following command:
+`helm uninstall <release name> -n <name space>`
+
+Example: 
+
+ >   ```
+  >   helm uninstall sap-btp-operator -n sap-btp-operator
+
+
+**Note:** All service bindings and service instances are also deleted during this process and the expected time out is up to 5 minutes.
+
+#### Possible Issues
+
+   - The deletion of instances and bindings takes more than 5 minutes
+   
+     There are two scenarios and their respective resolutions:
+     
+      - If you get a timeout error, manually re-trigger the delete process.
+      
+      - If you get the following error: `Warning: Hook pre-delete failed: jobs.batch "pre-delete-job" already exists.` please wait for the job to finish.
+     
+   - One of the service instances or bindings could not be deleted.
+     
+     Solution:
+     
+     Debug that resource issue and fix it before re-triggering the job.
```

Comment:
not all users have access to view jobs on the cluster, another solution to debug is to view the instances and binding and check their status to see why the deletion failed.